docs/templates/nextflow/main.nf

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env nextflow
+
+// write help message here
+if (params.help) {
+	
+	    log.info"""
+	    ==============================================
+	    APAeval pilot benmarking pipeline
+	    ==============================================
+	    Usage:
+	    Run the pipeline with default parameters:
+	    nextflow run main.nf -profile docker --input samplesheet.csv
+	    Mandatory arguments:
+                --input		samplesheet
+	    Flags:
+                --help			Display this message
+	    """.stripIndent()
+
+	exit 1
+}
+
+// define parameters (add more parameters if necessary)
+if (params.input) { ch_input = file(params.input, checkIfExists: true) } else { exit 1, "Samplesheet file not specified!" }
+
+/*
+ * Check input samplesheet
+ */
+process CHECK_SAMPLESHEET {
+    tag "$samplesheet"
+    publishDir "${params.outdir}/pipeline_info", mode: params.publish_dir_mode
+
+    input:
+    path samplesheet from ch_input
+    
+    output:
+    path "*reformat.csv" into ch_samplesheet_reformat
+    
+    script:
+    """
+    check_samplesheet.py $samplesheet samplesheet_reformat.csv
+    """
+}
+
```

Comment:
Can you add a comment on what this function does and why it is necessary in the workflow?

Reply:
Yes, I added a comment for this process.